code/datums/controllers/action_controls.dm

```diff
@@ -972,8 +972,7 @@ var/datum/action_controller/actions
 			logTheThing(LOG_COMBAT, source, "tries to put \an [item] on [constructTarget(target,"combat")] at at [log_loc(target)].")
 			icon = item.icon
 			icon_state = item.icon_state
-			for(var/mob/O in AIviewers(owner))
-				O.show_message("<span class='alert'><B>[source] tries to put [item] on [target]!</B></span>", 1)
+			source.visible_message("<span class='alert'><B>[source] tries to put [item] on [target]!</B></span>", TRUE)
```

Comment:
What exactly is the TRUE supposed to be in these. Like it is this tries to show TRUE as a message to yourself.